### DISCUSSION TITLE
組み立てマニュアル Groveコネクタのはんだ付け

### DISCUSSION THREAD

**Discussion by shoko93** [algyan/XIAOGYAN#58] [General]
Issueに書くか迷いましたがひとまずこちらに投稿しておきます。
組み立てマニュアルの2.5.4. Groveコネクタのはんだ付けについて。
秋月で購入したＬ型のGroveコネクタの場合そのままだと↓の写真の通りにはならないので、少し迷いました。
![image](https://github.com/algyan/XIAOGYAN/assets/9836975/fcfac116-d6eb-42db-a4e6-6ea24b4b6fb8)
（私の場合はよくわからなかったので、L字の足を無理やりまっすぐにしました。。）

**Comment by matsujirushi** [comment]
Issue立てましたー
